Makefile

```
@@ -81,7 +84,31 @@ generate-embeddedyamls: pkg/subctl/operator/common/embeddedyamls/yamls.go
 pkg/subctl/operator/common/embeddedyamls/yamls.go: pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go $(shell find deploy/ -name "*.yaml") vendor/modules.txt
 	go generate pkg/subctl/operator/common/embeddedyamls/generate.go
 
-.PHONY: $(TARGETS) test validate build ci clean generate-embeddedyamls operator-image
+# generate-clientset generates the clientset for the Submariner APIs
+# It needs to be run when the Submariner APIs change
+generate-clientset:
+	go get k8s.io/code-generator@kubernetes-1.14.1
+	$${GOPATH}/pkg/k8s.io/code-generator@v0.0.0-20190311093542-50b561225d70/generate-groups.sh \
```

```suggestion
	$${GOPATH}/pkg/mod/k8s.io/code-generator@v0.0.0-20190311093542-50b561225d70/generate-groups.sh \
```

Shouldn't this be the path?

I ran `go get k8s.io/code-generator@kubernetes-1.14.1` inside `make shell` and see that it's inside the `mod` directory

---

It should, and I even had that fixed locally, but forgot to commit it...